### ISSUE TITLE
Serialization of null-prototype objects failing.

### ISSUE THREAD

**Issue by markw351** [awspilot/dynamodb-util#2]
Given an object such as below

```
{
  tags: [Object: null prototype] { leagues: [], players: [], teams: [] }
}
```

this will fail when serialized

```
TypeError: $value.hasOwnProperty is not a function
      at Function.DynamoUtil.stringify (/Users/mark/dev/markw351-serverless/node_modules/@awspilot/dynamodb-util/dist/awspilot-dynamodb-util.js:217:15)
      at Function.DynamoUtil.anormalizeItem (/Users/mark/dev/markw351-serverless/node_modules/@awspilot/dynamodb-util/dist/awspilot-dynamodb-util.js:137:30)
```

**Comment by markw351** [comment]
Issue resolved in PR #4

**Comment by adrianpraja** [comment]
Hi Mark, I'll have a look and publish new release these days, thanks

**Comment by adrianpraja** [comment]
Merged and published v1.1.4
